src/renderers/stateRenderer.js

```diff
@@ -5,6 +5,66 @@ const fs = require('fs');
 require('bootstrap');
 
 // START UTILITY FUNCTIONS
+
+function sortTable(n,tableId) {
+    let table, rows, switching, i, x, y, shouldSwitch, dir, switchcount = 0;
+    table = document.getElementById(tableId)
+    console.log(table);
+
+    switching = true;
+    // Set the sorting direction to ascending:
+    dir = "asc";
```

Comment:
Do we need that comment? How about renaming the variable to `sortingDirection` or something like that?

Reply:
And extract it into a `const`